docs/docs-content/byoos/capi-image-builder/build-image-vmware/airgap-build/rhel-capi-airgap.md

```diff
@@ -0,0 +1,529 @@
+
+    :::info
+
+    You can also use the following steps to make the image available in a VMware vSphere environment that is not
+    connected to the one you used for building the image.
+
+    :::
+
+21. Right-click the folder and select **Deploy OVF Template** to deploy a VM using the RHEL 8 OVA file that was built in
+    step **16** of this guide.
+
+22. In the **Deploy OVF Template** wizard, click **Local File**, then **Upload Files**, and select the
+    `rhel-8-kube-v1.28.9.ova` file from the folder on your local machine. Click **Next** to continue.
+
+23. Assign a name to the virtual machine, such as `rhel-8-kube-v1.28.9`, and select the folder you created previously as
+    the target location. Click **Next** to proceed.
+
+24. Select a compute resource and click **Next**.
+
+25. Review the VM configuration, accept the license agreements, and click **Next**.
+
+26. Select the storage location and network configuration and click **Next**. Then, click **Finish** to deploy the VM.
+
+    :::warning
+
+    It takes a while for the VM to deploy, approximately 45 minutes or more. The download of the OVA file takes up the
+    majority of the time. You can monitor the progress of this process in VMware vSphere by looking at the **Recent
+    Tasks** tab and filtering the **Task Name** column by `Deploy OVF Template`.
+
+    :::
+
+27. Once the VM is created, right-click it and select **Convert to Template**. This will convert the VM into a RHEL 8
+    image template that you can reference during the cluster profile creation.
+
+## Create Cluster Profile
+
+The RHEL 8 image is now built and available in the VMware vSphere environment. You can use it to create a cluster
+profile and deploy a VMware vSphere host cluster.
+
+19. Log in to your airgapped instance of Palette or VerteX and select **Profiles** from the left **Main Menu**.
+
+20. Click **Add Cluster Profile** and follow the wizard to create a new profile.
+
+21. In the **Basic Information** section, assign the cluster profile a name and a brief description, and select the type
+    as **Full**. Click **Next**.
+
+22. In the **Cloud Type** section, select **VMware** and click **Next**.
+
+23. The **Profile Layers** section is where you specify the packs that compose the profile. This guide uses the
+    following packs as an example.
+
+    | Pack Name                   | Version | Layer            |
+    | --------------------------- | ------- | ---------------- |
+    | BYOOS                       | 1.0.0   | Operating System |
+    | Palette eXtended Kubernetes | 1.28.9  | Kubernetes       |
+    | Calico                      | 3.28.0  | Network          |
+    | vSphere CSI                 | 3.2.0   | Storage          |
+
+    <!-- prettier-ignore -->
+    Reference the custom RHEL 8 image template path in your VMware vSphere environment when populating the pack details
+    for the <VersionedLink text="BYOOS" url="/integrations/packs/?pack=generic-byoi" /> layer.
```

Comment:
```suggestion
    Reference the custom RHEL 8 image template path in your VMware vSphere environment when populating the pack details
    for the <VersionedLink text="BYOOS" url="/integrations/packs/?pack=generic-byoi" /> layer. For example, in the code snippet below, `/Datacenter/vm/sp-docs/rhel-8-kube-v1.28.9` is the vSphere path to the image.
```

Just minor suggestion to help the reader. People often copy and paste, so if we can help them, the better.